script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -559,12 +557,12 @@ You should be thanking us, Shiki-kun. You were this close to being turned into a
 No. That can't be right.
  It's true that I was afraid.
  That's an undeniable fact.
- But %{i}that%{/i} was definitely not going to happen.
+ But she never forced me to do anything.
```

Comment:
Yes

Reply:
Actually looking at it closer I see what you're saying.

But here's how I thought about it:

力に任せて人間を従わせる―――フン。(force a human to obey them)
He's responding to this.

けど、それだけは絶対にない。(but that's something she'd never do)
……一度もなかった。(she's never done it)
あいつはいつだって俺の選択を尊重していた。(she always did respect my choices)
力任せなんて、一度もなかったんだ。(forcing me [call back to the first line] hasn't happened once [callback to 一度もなかった])

So it's fairly clear that 一度もなかった。 is  力任せなんて、一度もなかったんだ。and then from the line between them, that it's specifically about him not being forced

that said, given how True Ancestors do 人間を従わせる by drinking blood, perhaps the this is in some ways 'splitting hairs'.

Reply:
I also think that けど、それだけは絶対にない。should be talking about the future. I agree that ...一度もなかった is clearly 力任せなんて、一度もなかったんだ, but I don't think its a continuation from the previous sentence but rather a natural progression in his thoughts.

> She'd never do it. けど、それだけは絶対にない。
> (Why? Because) She's never done it before. ……一度もなかった
> She's always respected my choices. あいつはいつだって俺の選択を尊重していた。
> She's never forced me to, not even once. 力任せなんて、一度もなかったんだ。

Reply:
The more I thought about this I think the issue is simply that 人間を従わせる is the same thing as turning someone into a vampire so Shiki isn't responding to one or the other, but both (or to be more precise, they're the same so there's no conflict in his thoughts at all)

So: 
-- 「力に任せて人間を従わせる―――フン。
"Forcing a human into serving you―――%{i}Hmph%{/i}.

-- いや。それは違う。俺が怯えたのは本当だ。弁明できない事実だ。けど、それだけは絶対にない。
No. That can't be right.
 It's true that I was afraid.
 That's an undeniable fact.
 But she would %{i}never%{/i} do that.
 
 -- ……一度もなかった。
It hasn't happened even once.

-- あいつはいつだって俺の選択を尊重していた。
She's always respected my choices.

-- 力任せなんて、一度もなかったんだ。
Her forcing me... hasn't happened even once.